crates/common_enums/src/enums.rs

```diff
@@ -1176,6 +1179,7 @@ pub enum IntentStatus {
     Processing,
     RequiresCustomerAction,
     RequiresMerchantAction,
+    FrmRequiresMerchantAction,
```

Comment:
we can use RequiredMerchantAction, this was meant to indicate that there is some MerhantAction that is required

Reply:
cc: @AnandKGanesh @bernard-eugine

Reply:
@Narayanbhat166 , We have introduced this status to not interfere with payments which doesn't go through FRM. In future there might decisions/actions we might take based on `FrmRequiredMerchantAction` status for FRM webhooks which doesn't apply for payment connectors.

cc: @kashif-m @AnandKGanesh

Reply:
Right, this status is introduced for explicitly specifying the fact that merchant needs to take an FRM related action on the payment. Using existing variant `RequiresMerchantAction` does not give enough context about what exactly merchant needs to do. This variant is being used in one of the flows for some crypto connector cc: @srujanchikke 

Having a separate variant in this case gives us full context about the type of actions that can be taken by the merchant (payment approval / cancellation in this case)

Reply:
cc : @jarnura

Reply:
We should not introduce the same status `RequiresMerchantAction` with some prefix again because the merchant wants to do something different by this. If `RequiresMerchantAction` is the status, then always the merchant should go to some details field or reason field, which tells what is issue and what merchant needs do.

Reply:
Let's also add the feasibility to be able to list only the fraudulent transactions in payments list / filter APIs using the details / reason field

Reply:
Sure, will do that it in subsequent PR